x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table_header.tsx

```diff
@@ -262,12 +273,18 @@ const CurrentPageOfTotal = ({
   refetchRulesStates,
   setSelectAllRules,
   setSelectedRules,
+  notifications,
```

Comment:
the notifications object is available globally for us to use through context within the `useKibana` hook, there are many places where doesn't need to pass down notifications multiple times across all the children component, instead we can get notifications from the last child component like this:

```js
 const { notifications } = useKibana().services;
```